metric_config_parser/metric.py

```diff
@@ -91,7 +91,7 @@ class Metric:
     type: str = "scalar"
     category: Optional[str] = None
     depends_on: Optional[List[Summary]] = None
-    owner: Optional[str] = None
+    owner: Optional[List[str]] = None
```

Comment:
This reminded me of the python/ruff updates in mozanalysis.. perhaps we should update m-c-p's python version to bring it and mozanalysis and jetstream inline with each other.

Reply:
sure!

Reply:
for posterity: https://github.com/mozilla/metric-config-parser/issues/360